numpy/add_newdocs.py

```diff
@@ -463,6 +463,47 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create a outermost-first tuple of iterators. They are nested so that
+    advancing one will change the inner iterators to point at the new
+    element in the outer iterator.
+
+    Parameters 
+    ----------
+    op : ndarray
+        The array to iterate over.
```

Comment:
Sure this doesn't work for multiple arguments?

Reply:
turns out it _does_ work for multiple input args as long as the values in op have the same shape for the values in axes